src/main/java/randoop/util/WeightedList.java

```diff
@@ -2,19 +2,25 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import randoop.BugInRandoopException;
 
 public class WeightedList {
 
   private List<WeightedElement> theList;
+  private List<Double> cumulativeWeights;
```

Comment:
Please document the new fields you added.  It would be a good deed to add documentation elsewhere, especially if that would help in understanding your changes.